_patterns/synths.md

```diff
@@ -24,8 +24,22 @@ The above plays note 60 and 62 of the MIDI scale, using the `midinote` parameter
 ~~~haskell
 d1 $ n "c5 d5*2" # s "supersaw"
 ~~~
+
+For half tones you add the suffixes "f" or "s" (flat or sharp) to the note in question. 
+
+~~~haskell
+d1 $ n "<[a5,cs5,e5,g5]*3 [d5,fs5,g5,c5]>" # s "supersquare" # gain "0.7"
+~~~
 {: .render}
 
+Above is a two chord progression A7 D7. Notice `cs5` and `fs5` as C5# and F5#, respectively.
+
```

Comment:
Do we want the sharps to be before or after the octave number here?  I guess I would naturally think it should go C#5 rather than C5#, but I don't know if one or the other is more standard.